plugin/session_view.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, listener: AbstractViewListener, session: Session) -> None:
         self._code_lenses = CodeLensView(self.view)
         settings = self.view.settings()
         buffer_id = self.view.buffer_id()
-        key = (session.config.name, session.window.id(), buffer_id)
+        key = (id(session), session.window.id(), buffer_id)
```

Comment:
Since `session` id is so unique, we probably don't need the `window.id()` in the tuple anymore?